systems/plants/centroidalMomentumMatrixmex.cpp

```diff
@@ -0,0 +1,49 @@
+#include <mex.h>
+#include <iostream>
+#include <memory>
+#include "drakeUtil.h"
+#include "RigidBodyManipulator.h"
+#include "math.h"
+
+using namespace Eigen;
+using namespace std;
+
+// TODO: stop copying these functions everywhere and find a good place for them
```

Comment:
Yeah, so where should these functions live? Also, I believe they can be made a bit more efficient.